src/navbar/brand.tsx

```diff
@@ -29,24 +29,41 @@ export const Brand = ({
     // =============================================================================
     const handleClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
         if (onClick) {
+            event.preventDefault();
             onClick(event, type);
         }
     };
 
+    // Determine if this should be clickable - brands are clickable by default when onClick is provided
+    const isClickable = !!onClick;
+
+    const ariaLabel =
+        type === "primary"
+            ? `Go to ${resources.brandName} homepage`
+            : `Go to ${resources.brandName}`;
+
+    if (isClickable) {
+        return (
+            <Clickable
+                data-id={dataId}
+                data-testid={testId}
+                $type={type}
+                href="#"
```

Comment:
don't set the href (it becomes a functioning link that might not reflect the right url or visited state), continue to set a `role="link"` to prevent Lighthouse from complaining about SEO

planning to audit the usage of anchor tags and expose href props across the navigation elements in a future enhancement